user-access-service/src/test/java/no/unit/nva/database/IdentityServiceTest.java

```diff
@@ -84,24 +84,6 @@ public void init() {
         usersTable = enhancedClient.table(IdentityService.USERS_AND_ROLES_TABLE, UserDao.TABLE_SCHEMA);
     }
 
-    @Test
```

Comment:
only reformating of the file and renaming the method `getUserByCristinIdAndCristinOrgId` to `getUserByPersonCristinIdAndCustomerCristinId`